src/input_xml.F90

```diff
@@ -2352,8 +2355,10 @@ subroutine read_tallies_xml()
 
           ! Determine number of bins
           if (check_for_node(node_filt, "bins")) then
-            if (trim(temp_str) == 'energy' .or. &
-                 trim(temp_str) == 'energyout') then
+            if ((trim(temp_str) == 'energy' .or. &
+                 trim(temp_str) == 'energyout') .or. &
+                 (trim(temp_str) == 'mu' .or. trim(temp_str) == 'polar') .or. &
+                 (trim(temp_str) == 'azimuthal')) then
```

Comment:
Per the Fortran standard, string comparisons on two un-equal length strings treat the shorter one as though it were space-padded to the same length as the longer. Thus, `trim` is not necessary here, and removing it would make this look a little prettier.